packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -681,6 +846,17 @@ export class UserPool extends Resource implements IUserPool {
     });
   }
 
+  /**
+   * Set risk Configuration for this user pool.
+   * @see https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-advanced-security.html
+   */
+  public setUserPoolRiskConfiguration(id: string, options: UserPoolRiskConfigurationOptions): CfnUserPoolRiskConfigurationAttachment {
+    return new CfnUserPoolRiskConfigurationAttachment(this, id, {
+      userPoolId: this.userPoolId,
+      ...options,
+    });
+  }
```

Comment:
Separate this into its own construct called `UserPoolRiskConfigurationAttachment` and into a separate file called `user-pool-security.ts`. Move all associated interfaces into this file as well.